### ISSUE TITLE
Notofications scope

### ISSUE THREAD

**Issue by alec-c4** [excid3/noticed#119]
Hey!
Is it possible to separate notifications by scope? for example - post and comments notifications for users and technical notifications for admins

**Comment by excid3** [comment]
Can you share an example of what you're thinking?

**Comment by alec-c4** [comment]
of course, I'll create separate notifications for each event, but how to separate them when querying?

**Comment by alec-c4** [comment]
I mean the same model (User), but different roles (I use rolify gem).

**Comment by excid3** [comment]
Probably need to join that into your query then.
